src/modelfree/common/utils.py

```diff
@@ -149,39 +155,114 @@ def _reset_video_recorder(self, metadata=None):
         )
 
 
+class ReporterOutputFormat(KVWriter):
```

Comment:
Why is this here? It's also in `src/modelfree/multi/train_worker.py`, which is the only place I can see it being used. (Would be reasonable to factor out -- although probably not into `util` -- if it was used in multiple places.)